File: 程式語言 C V2答案檔/301.c
#include <stdio.h>
#include <stdlib.h>

char compute(int s[]);
int main()
{
    int box[2];
    scanf("%d", &box[0]);
    scanf("%d", &box[1]);
    printf("%d\n", compute(box));
    return 0;
}

// ===================================================
// Please write your code in the specified function
// Do NOT change the function name, type of parameter
// ===================================================
char compute(int box[])
{
    int x, y;
    for (y = 0; y < box[1]; y++)
    {
        for (x = 0; x < box[0]; x++)
        {
            printf("*");
        }
        printf("\n");
    }
    return box[0] * box[1];
}

File: 物件導向程式語言 C# 7答案檔/301.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CSA03
{
    class CSA03
    {
        static void Main(string[] args)
        {
            int[] numbers = new int[5];
            // TODO
            float number;
            for (int i = 0; i < numbers.Length; i++)
            {
                float.TryParse(Console.ReadLine(), out number);
                numbers[i] = (int)number;
            }
            compute(numbers);
        }

        // ==================================================
        // Please write your code in the specified Method
        // Do NOT change the method name, type of parameter
        // ==================================================
        static void compute(int[] numbers)
        {
            int sum = 0; int times = 0;
            //TODO
            for (int i = 0; i < numbers.Length; i++)
            {
                if (numbers[i] > 50)
                {
                    sum += numbers[i];
                    times++;
                }
            }

            Console.WriteLine(sum.ToString());
            if (times > 0)
            {
                Console.WriteLine((sum / times).ToString());
            }
            else
            {
                Console.WriteLine("0");
            }
            Console.ReadKey();
        }


    }
}
